community/plugins/spring-ai-alibaba-starter-plugin-larksuite/src/main/java/com/alibaba/cloud/ai/plugin/larksuite/LarkSuiteAutoConfiguration.java

```diff
@@ -26,16 +26,23 @@
 /**
  * @author 北极星
  */
-@EnableConfigurationProperties({ LarkSuiteProperties.class })
-@ConditionalOnClass({ LarkSuiteProperties.class })
+@EnableConfigurationProperties({LarkSuiteProperties.class})
+@ConditionalOnClass({LarkSuiteProperties.class})
 public class LarkSuiteAutoConfiguration {
 
-	@Bean
-	@ConditionalOnMissingBean
-	@Description("it calls the document api to invoke a method to create a larksuite document")
-	@ConditionalOnProperty(prefix = "spring.ai.alibaba.plugin.larksuite", name = "enabled", havingValue = "true")
-	public LarkSuiteService larksuiteCreateDocFunction(LarkSuiteProperties properties) {
-		return new LarkSuiteService(properties);
-	}
+    @Bean
+    @ConditionalOnMissingBean
+    @Description("it calls the document api to invoke a method to create a larksuite document")
+    @ConditionalOnProperty(prefix = "spring.ai.alibaba.plugin.larksuite", name = "enabled", havingValue = "true")
+    public LarkSuiteDocService larksuiteCreateDocFunction(LarkSuiteProperties properties) {
+        return new LarkSuiteDocService(properties);
+    }
 
+    @Bean
+    @ConditionalOnMissingBean
+    @Description("it runs a api to invoke a method to send group message")
```

Comment:
Do we need to add `larksuite` keyword in the description for LLM to better understand that this is a Larksuite messaging tool?

Reply:
yes ,of course ,each proxy bean have different obligatory